en-US/Docs/SetupWiFiDirect.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: docs
+title: Using WiFi Direct on your Windows 10 IoT Core device
+description: Learn how to use Wifi Direct on your Windows 10 IoT Core device
+keyword: wifi, wifi direct, windows iot, device, wireless
+permalink: /en-US/Docs/SetupWiFiDirect.htm
+lang: en-US
+---
+
+# Using WiFi Direct on your Windows 10 IoT Core device
+
+WiFi Direct is supported on Windows 10 IoT Core devices through the use of a WiFi Direct enabled USB WiFi adapter. To make sure that WiFi Direct is enabled two things need to be true:
```

Comment:
Comma after enabled (To make sure that WiFi Direct is enabled, following two requirements need to be met)